### PR TITLE
draft changes for v1.1

### PR DESCRIPTION
**Pull request by m-burgoyne**
First draft of proposed updates proposed for v1.1
    - Support for HTTP Post
    - Support for custom dimensions

Includes refactoring of the documentation of data query patterns, the information is now provided as tables

### CODE REVIEW

File: standard/requirements/edr/REQ_rc-point-coords-response.adoc
Comment (iandruska-ibl) [inline standard/requirements/edr/REQ_rc-point-coords-response.adoc:19]: I believe that HTTP 405 is reserved strictly for HTTP method (GET, HEAD, POST, PUT, ...) mismatch, i.e. client used an HTTP method which is not supported by the endpoint.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405

File: standard/requirements/requirements_class_custom_dimensions.adoc
Comment (iandruska-ibl) [inline standard/requirements/requirements_class_custom_dimensions.adoc:20]: I believe this should be
```
minItems: 2
maxItems: 2
```

File: standard/requirements/requirements_class_custom_dimensions.adoc
Comment (iandruska-ibl) [inline standard/requirements/requirements_class_custom_dimensions.adoc:22]: wrong indentation of `anyOf:`

**Review by iandruska-ibl — CHANGES_REQUESTED** [review]

**Review by dblodgett-usgs — APPROVED** [review]
I see a couple empty files -- should those be in here? 

It's too bad that we have to copy/paste so much content to add the POST method to all the queries -- seems like there should have been some way to be less repetitive. meh.

Is the "limit" requirement not satisfied by API Features?

I'm going to want to see some of this stuff rendered before knowing whether it's right or not.

**Review by chris-little — COMMENTED** [review]
A lot of minor editorial typos, originally corrected by @ghobona have crept back in. I thought there were only a few rather than 81, so did in place on the branch.
Examples:
a `f` -> an `f`
seperate -> separate (Always A RAT in separate)
missing right parentheses
end point -> endpoint

There were also lots of GitHub warning messages about files not ending with a Newline.

Ther trajectory example needs fixing - it slides into being a corridor example.

File: standard/requirements/edr/REQ_rc-point-coords-response.adoc
Comment (chris-little) [reply standard/requirements/edr/REQ_rc-point-coords-response.adoc]: In PR#390

File: standard/openapi/request-bodies/corridor.yaml
Comment (chris-little) [reply standard/openapi/request-bodies/corridor.yaml:6]: In PR#390

File: standard/requirements/requirements_class_custom_dimensions.adoc
Comment (chris-little) [reply standard/requirements/requirements_class_custom_dimensions.adoc]: In PR#390, but not `maxItems: 2`

File: standard/requirements/requirements_class_custom_dimensions.adoc
Comment (chris-little) [reply standard/requirements/requirements_class_custom_dimensions.adoc]: In PR#390

**Comment by chris-little** [comment]
@m-burgoyne can you fix the trajectory/corridor problem?

@dblodgett-usgs Then we can merge and have some text to review in detail

**Comment by chris-little** [comment]
@dblodgett-usgs @m-burgoyne @ShaneMill1 @solson-nws @iandruska-ibl @tomkralidis I propose to not wait for more review approvals, so we can get a first cut at the V1.1document

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**
